site/templates/nav.jet

```diff
@@ -66,5 +66,6 @@
       </nav>
     {{end}}
   </div>
+  <div class="tablet-overlay"></div>
```

Comment:
I think a better class would be `s72-tablet-backdrop` to match our existing stuff.